modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1088,8 +1088,13 @@ protected int processStartTag() throws Exception {
 				jspWriter.write("\"");
 			}
 
+			String searchPlaceholderKey =
+				FeatureFlagManagerUtil.isEnabled("LPD-11313") ? "search" :
+					"search-for";
+
```

Comment:
This is only used once. Can you inline it?

Reply:
Thanks for reviewing this @adolfopa. I've inlined it in https://github.com/liferay-content-management/liferay-portal/pull/5297/commits/1841ff65bca023e27e6fd9d2341f31dfba987a5a